packages/react/src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactNode } from 'react';
+import Button from 'devextreme-react/button';
+import Toolbar, { Item } from 'devextreme-react/toolbar';
+
+type DashboardContainerProps = {
+  title: string;
+  additionalToolbarContent?: ReactNode;
+};
+
+export const Dashboard = ({ title, additionalToolbarContent, children }: React.PropsWithChildren<DashboardContainerProps>) => {
+  return (
+    <div className='view-wrapper-dashboard'>
+      <Toolbar>
+        <Item location='before'>
+          <span className='toolbar-header'>{title}</span>
+        </Item>
+        {additionalToolbarContent}
+        <Item location='after' locateInMenu='auto'>
+          <Button className='add-card' icon='plus' text='Add Card' type='default' stylingMode='contained' />
```

Comment:
Same problem about "Difficult read all props"

Reply:
modified prettier settings to prevent long lines